var/spack/repos/builtin/packages/adios/package.py

```diff
@@ -12,30 +36,107 @@ class Adios(Package):
     """
 
     homepage = "http://www.olcf.ornl.gov/center-projects/adios/"
-    url = "https://github.com/ornladios/ADIOS/archive/v1.9.0.tar.gz"
+    url      = "https://github.com/ornladios/ADIOS/archive/v1.10.0.tar.gz"
+
+    # make -j is currently not supported
+    parallel = False
+
+    version('1.10.0', 'eff450a4c0130479417cfd63186957f3')
+    version('1.9.0' , '310ff02388bbaa2b1c1710ee970b5678')
+    version('1.8.0' , 'c9116ec31e6b386f0e6ea5ab675c57e9')
+    version('1.7.0' , '266897ee3a390985d840bbe2935b4293')
+    version('1.6.0' , 'b3c735f3afbf552ec0d5e8152bffd778')
+
+    variant('shared', default=True,
+            description='Builds a shared version of the library')
 
-    version('1.9.0', '310ff02388bbaa2b1c1710ee970b5678')
+    variant('fortran', default=True,
+            description='Enable Fortran bindings support')
+
+    variant('mpi', default=False, description='Enable MPI support')
+    variant('infiniband', default=False, description='Enable infiniband support')
+
+    variant('zlib', default=True, description='Enable szip transform support')
+    variant('szip', default=False, description='Enable szip transform support')
+    variant('hdf5', default=False, description='Enable HDF5 transport support')
+    variant('netcdf', default=False, description='Enable NetCDF transport support')
 
     # Lots of setting up here for this package
     # module swap PrgEnv-intel PrgEnv-$COMP
     # module load cray-netcdf/4.3.3.1
     # module load cray-hdf5/1.8.14
     # module load python/2.7.10
-    depends_on('hdf5')
-    depends_on('mxml')
+
+    depends_on('autoconf')
```

Comment:
Please remove mxml from the dependencies. adios 1.10 can build it's own now.

On Mon, Aug 29, 2016 at 2:23 PM, Denis Davydov <notifications@github.com>
wrote:

> In var/spack/repos/builtin/packages/adios/package.py
> <https://github.com/LLNL/spack/pull/1652#discussion_r76659278>:
>
> >
> >      # Lots of setting up here for this package
> >      # module swap PrgEnv-intel PrgEnv-$COMP
> >      # module load cray-netcdf/4.3.3.1
> >      # module load cray-hdf5/1.8.14
> >      # module load python/2.7.10
> > -    depends_on('hdf5')
> > -    depends_on('mxml')
> > +
> > +    depends_on('autoconf')
>
> this and next one are build-only dependencies, right? If so, please
> declare as such.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1652/files/e1464ece55198065b7801e18dfedc9eab2e07286#r76659278>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADGMLeZjaAryK_W_qNZRVfkpKF9JDrIXks5qkyOigaJpZM4Jvkjc>
> .
>